### PR TITLE
Option to enable service

### PR DESCRIPTION
**Pull request by akurdyukov**
Please add option to enable service on the system

### CODE REVIEW

**Comment by otakup0pe** [comment]
If you can update the README I'll merge this

**Comment by akurdyukov** [comment]
I updated the README, please check

File: defaults/main.yml [hunk 1]
```diff
@@ -14,6 +14,7 @@ postgresql_default_auth_method: "trust"
 # The user/group that will run postgresql process or service
 postgresql_service_user: "{{ postgresql_admin_user }}"
 postgresql_service_group: "{{ postgresql_admin_user }}"
+postgresql_service_enabled: false
```
Comment (UnderGreen) [inline defaults/main.yml]: Why postgresql service disabled by default? Service should be enabled by default. And in README you wrote enabled default value.

Comment (akurdyukov) [inline defaults/main.yml]: Because it was disabled by default. I decided not to break behaviour. Do you think it's reasonable to enable by default?

Comment (UnderGreen) [inline defaults/main.yml]: I don't know about RH-based systems, but in Deb-based systems - postgresql service enabled.

Comment (otakup0pe) [inline defaults/main.yml]: The difference is probably due to how the package is installed on the different OS flavors.

I'd say enable it by default and we'll bump the major version component when we merge :)

Comment (UnderGreen) [inline defaults/main.yml]: :+1:

Comment (akurdyukov) [inline defaults/main.yml]: Switched default to true, please check.

**Pull request merged**